editor/editor_audio_buses.h

```diff
@@ -87,6 +87,7 @@ class EditorAudioBus : public PanelContainer {
 	Tree *effects = nullptr;
 
 	bool updating_bus = false;
+	bool renaming_bus = false;
```

Comment:
Unused?

Reply:
Ah, yes. Used to be used before `renaming_buses` in the main class was introduced.

Reply:
Compiler warnings are supposed to flag this kind of declared but unused stuff, but somehow they don't.

Reply:
That would be nice to have working 👀